protocol-units/da-sequencer/node/src/celestia/client.rs

```diff
@@ -1,29 +1,25 @@
 use super::submit::BlobSubmitter;
-use super::{
-	BlockSource, CelestiaBlobData, CelestiaClientOps, CelestiaHeight, ExternalDaNotification,
-};
-use crate::block::SequencerBlockDigest;
+use super::{BlockSource, CelestiaBlob, CelestiaClientOps, CelestiaHeight, ExternalDaNotification};
 use crate::error::DaSequencerError;
-
 use celestia_rpc::Client as RpcClient;
 use celestia_types::nmt::Namespace;
+use movement_types::block;
+use std::sync::Arc;
 use tokio::sync::mpsc;
 use url::Url;
 
-use std::sync::Arc;
-
 #[derive(Debug, thiserror::Error)]
-enum Error {
+pub enum Error {
 	#[error("Celestia RPC error: {}", .0)]
 	Rpc(#[from] celestia_rpc::Error),
 }
 
 #[derive(Clone)]
 pub struct CelestiaClient {
-	rpc_client: Arc<RpcClient>,
-	notifier: mpsc::Sender<ExternalDaNotification>,
+	_rpc_client: Arc<RpcClient>,
+	_notifier: mpsc::Sender<ExternalDaNotification>,
```

Comment:
These will come useful when `get_blob_at_height` is implemented (in a separate upcoming PR). But I can deal with it.

Reply:
It's just temporary to get rid of the compiler warnings.

Reply:
In general for feature branches we are relaxed on them, only when merged feature branch into main do we want to reduce the warnings. In some ways we want to to see the compiler warnings when working on the feature branch, as they will become todos before we merge into main. Otherwise, we can miss them. But this is fine for now.

Reply:
The noise generated by warnings in this feature branch was massive. I had difficulty locating compile errors among them and had to scroll back and forth to identify them.